src/components/sendReadable/send.js

```diff
@@ -79,13 +79,15 @@ class SendReadable extends React.Component {
   send(event) {
     event.preventDefault();
     this.setState({ loading: true });
+    const isAccountInit = this.props.accountInit &&
+      (this.props.account.address === this.state.recipient.value);
```

Comment:
The second line of the condition is not needed, `this.props.accountInit` should be already all you need.